addons/medical_engine/functions/fnc_handleDamage.sqf

```diff
@@ -200,6 +200,11 @@ if (
     0
 };
 
+// No need to go any further if vehicle crash damage is disabled
+if (!EGVAR(medical,enableVehicleCrashes)) exitWith {
```

Comment:
Would prefer if this condition was just part of the vehicle crash `if`'s condition.

Reply:
I need to start drinking more coffee when doing these.